File: poo/Java/Bissexto/Bissexto.java
import java.util.Scanner;

public class Bissexto {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int ano = scanner.nextInt();
        if ((ano % 4 == 0 && ano % 100 != 0) || (ano % 400 == 0)) {
            System.out.println("ANO BISSEXTO");
        } else {
            System.out.println("ANO NAO BISSEXTO");
        }
        scanner.close();
    }
}

File: poo/Cpp/Bissexto/Bissexto.cpp
#include <iostream>

int main() {
    int ano;
    std::cin >> ano;
    if ((ano % 4 == 0 && ano % 100 != 0) || (ano % 400 == 0)) {
        std::cout << "ANO BISSEXTO" << std::endl;
    } else {
        std::cout << "ANO NAO BISSEXTO" << std::endl;
    }
    return 0;
}